README.md

```diff
@@ -216,6 +216,25 @@ project-name
 ```
 and also repository with one commit at master will be added with enabled `Travis-CI` for that.
 
+## Warnings
+
+`ghc-options` field has default warnings in the  _project-name.cabal_:
```

Comment:
Strictly speaking, not all of these are warnings. It's better to tell something like this:

The following warning checks are added by default to every stanza:

-Wall

The following `ghc-options` are added to `executable`, `test-suite` and `benchmark` stanza.

-threaded and other friends

Also, you need to update part of README regarding `tomland` configuration and tell users that there's `warnings` field in toml file. And that users can override set of warnings through configuration.